onnx/defs/math/defs.cc

```diff
@@ -198,6 +198,23 @@ output data (Tensor<T>) where the function `f(x) = alpha * x for x < 0`,
     .TypeConstraint("T", { "tensor(float16)", "tensor(float)", "tensor(double)" },
         "Constrain input and output types to float tensors.");
 
+OPERATOR_SCHEMA(ThresholdedRelu)
```

Comment:
Again, should it be Relu with parameter? (and backends and bail out if they don't support it)